### PR TITLE
[WIP] Add a README to project

### PR DESCRIPTION
**Pull request by fwfurtado**

### CODE REVIEW

**Comment by pyumii** [comment]
ok

**Review by pyumii — DISMISSED** [review]

File: tests/loja/factories/product_factory.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import factory
+from loja.models.product import Product
+
+
+class ProductFactory(factory.Factory):
+    class Meta:
+        model = Product
+
+    name = factory.Faker('license_plate')
+    value = factory.Faker('pyfloat')
```
Comment (tayumii) [inline tests/loja/factories/product_factory.py]: 'value' -> change to 'price'

File: tests/loja/models/test_order.py [hunk 2]
```diff
@@ -3,12 +3,15 @@
...
+from tests.loja.factories.product_factory import ProductFactory
+
 
 class TestOrder:
 
     def test_should_compute_total_from_items(self):
-        t_shirt = Product("TShirt", 15)
-        shorts = Product("Shorts", 25)
+
+        t_shirt = ProductFactory.create(value=15)
```
Comment (tayumii) [inline tests/loja/models/test_order.py]: 'value' to 'price'

**Review by massaoozono — APPROVED** [review]

**Review by massaoozono — APPROVED** [review]

**Review by massaoozono — DISMISSED** [review]

**Review by pyumii — APPROVED** [review]
ok

**Review by tayumii — APPROVED** [review]
super top

**Pull request merged**